grails-app/views/index.gsp

```diff
@@ -2,76 +2,84 @@
-        <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Installed Plugins <span class="caret"></span></a>
-        <ul class="dropdown-menu">
-            <g:each var="plugin" in="${applicationContext.getBean('pluginManager').allPlugins}">
-                <li class="dropdown-item"><a href="#">${plugin.name} - ${plugin.version}</a></li>
-            </g:each>
-        </ul>
-    </li>
-</content>
-
-<div class="svg" role="presentation">
-    <div class="grails-logo-container">
-        <asset:image src="grails-cupsonly-logo-white.svg" class="grails-logo"/>
-    </div>
-</div>
-
-<div id="content" role="main">
-    <section class="row colset-2-its">
-        <h1>Welcome to Grails</h1>
-
-        <p>
-            Congratulations, you have successfully started your first Grails application! At the moment
-            this is the default page, feel free to modify it to either redirect to a controller or display
-            whatever content you may choose. Below is a list of controllers that are currently deployed in
-            this application, click on each to execute its default action:
-        </p>
-
-        <div id="controllers" role="navigation">
-            <h2>Available Controllers:</h2>
-            <ul>
-                <g:each var="c" in="${grailsApplication.controllerClasses.sort { it.fullName } }">
-                    <li class="controller">
-                        <g:link controller="${c.logicalPropertyName}">${c.fullName}</g:link>
-                    </li>
-                </g:each>
-            </ul>
-        </div>
-    </section>
-</div>
-
+	<main class="container">
+		<h1 class="title">Cadastro</h1>
+		<p id="error"> </p>
+		<form id="form-cadastro">
+			<div class="row">
+				<div class="inputbox">
+					<input type="text" id="nome" autocomplete="off" required>
+					<label for="nome">Nome</label>
+				</div>
+				<div class="inputbox">
+					<input type="text" id="cpf-cnpj" minlength="11" maxlength="14" autocomplete="off" required>
+					<label for="cpf-cnpj">CPF/CNPJ</label>
+				</div>
+			</div>
+			<div class="row">
+				<div class="inputbox">
+					<input type="email" id="email" autocomplete="off" required>
+					<label for="email">Email</label>
+				</div>
+                <div class="inputbox">
+					<input type="text" id="telefone" minlength="11" maxlength="11" autocomplete="off" required>
+					<label for="telefone">Telefone</label>
+				</div>
+			</div>
+			<div class="row">
+                <div class="inputbox">
+					<input type="text" id="cep" minlength="8" maxlength="8" autocomplete="off" required>
+					<label for="cep">CEP</label>
+				</div>
+                <div class="inputbox">
+					<input type="text" id="cidade" autocomplete="off" required>
+					<label for="cidade">Cidade</label>
+				</div>
+			</div>
+			<div class="row">
+				<div class="inputbox">
+					<input type="text" id="estado" maxlength="2" autocomplete="off" required>
```

Comment:
quando for feito o backend eu gostaria de ver esta regra de "maxlength" sendo buscada do backend, é um bom ponto pra vc já deixar anotado de lado e verificar como fazer quando finalizar o backend.